### ISSUE TITLE
Spring Boot support

### ISSUE THREAD

**Issue by michael-conway** [irods-contrib/metalnx-web#32]
Allow MetaLnx to run as a self-contained Spring Boot application, eliminating the need for Tomcat or other container

**Comment by michael-conway** [comment]
see issue branch 32-spring-boot

**Comment by trel** [comment]
is this being merged before 2.0.0?

**Comment by michael-conway** [comment]
no, that's more work, there are some initial things out there that are fine
to be in 2.0.0. They have no effect until this is done.

**Comment by trel** [comment]
Got it - marking as post 2.0.0
